src/convertToObject.js

```diff
@@ -5,8 +5,26 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const stylesArray = stylesString.split(';');
+  const stylesObject = {};
+
+  for (let i = 0; i < stylesArray.length; i++) {
+    const parts = stylesArray[i].split(':');
+
+    if (parts[0] && parts[1]) {
+      const property = parts[0].trim();
+      let value = parts[1].trim();
+
+      if (value.includes('!important')) {
+        value = value.replace(/\s*!important\s*/, '') + ' !important';
```

Comment:
[CODE KNOWLEDGE] - It's not necessary to handle '!important' separately unless specified. If it is part of the value, it will naturally be included when you trim and assign the value to the property. You can simplify the code by removing the conditional check for '!important'.